src/lightning_app/cli/lightning_cli.py

```diff
@@ -29,10 +31,68 @@ def get_app_url(runtime_type: RuntimeType, *args) -> str:
 @click.group()
 @click.version_option(ver)
 def main():
-    register_all_external_components()
     pass
 
 
+@main.group()
+def show():
+    """Show info about resource."""
+    pass
+
+
+@show.command()
+@click.argument("app_name", required=False)
```

Comment:
this should be required? 

should we also allow using `app_id`?

Reply:
We can think about using `app_id` that's a good idea actually... 
It's `required=False` because later we do a check anyway, and for better UX we list all available applications.  Pushing check into the `click` validator would mess up the code ...

If we do not provide the `app_name` we get something like 
```
Error: You have not specified any LightningApp. Please select one of available: [brawny-cerf-2556, lightning/hackernews-app-0.0.2]
```